src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -359,45 +482,53 @@ function ComposerWithSuggestions({
 
     /**
      * Callback to add whatever text is chosen into the main input (used f.e as callback for the emoji picker)
-     * @param {String} text
      */
     const replaceSelectionWithText = useCallback(
-        (text) => {
+        (text: string) => {
             updateComment(ComposerUtils.insertText(commentRef.current, selection, text));
         },
         [selection, updateComment],
     );
 
     const triggerHotkeyActions = useCallback(
-        (e) => {
-            if (!e || ComposerUtils.canSkipTriggerHotkeys(isSmallScreenWidth, isKeyboardShown)) {
+        (event: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
+            const webEvent = event as unknown as KeyboardEvent;
+            if (!webEvent || ComposerUtils.canSkipTriggerHotkeys(isSmallScreenWidth, isKeyboardShown)) {
                 return;
             }
 
-            if (suggestionsRef.current.triggerHotkeyActions(e)) {
+            if (suggestionsRef.current?.triggerHotkeyActions(webEvent)) {
                 return;
             }
 
             // Submit the form when Enter is pressed
-            if (e.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey && !e.shiftKey) {
-                e.preventDefault();
+            if (webEvent.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey && !webEvent.shiftKey) {
+                webEvent.preventDefault();
                 handleSendMessage();
             }
 
             // Trigger the edit box for last sent message if ArrowUp is pressed and the comment is empty and Chronos is not in the participants
             const valueLength = valueRef.current.length;
-            if (e.key === CONST.KEYBOARD_SHORTCUTS.ARROW_UP.shortcutKey && textInputRef.current.selectionStart === 0 && valueLength === 0 && !includeChronos) {
```

Comment:
I am not able to test this change because Chronos is available for `@expensify` mails , but it looks like it is using `includeChronos` instead of `includesChronos` so we should be good